_articles/we-dedicate-5-of-our-gross-revenue-to-environmental-causes-and-open-source-software.md

```diff
@@ -0,0 +1,86 @@
+
+We've started this commitment on January 1st, 2021. We're allocating 5% of our gross revenue into a separate account every month. We'll distribute and donate 100% of what we have in that account at some point before the end of this year.
+
+As usual with Plausible, we will do it all with full transparency and openness to our supporters. We will pick a mix of environmental causes and open source software to donate to. And we will announce it all here.
+
+### The environmental causes 
+
+Plausible Analytics has always been a project very conscious about how we as humans impact our planet and the environment. We're 100% powered by renewable energy and we've taken many steps to be as lightweight as possible both on our own website, our stats dashboard and the script that people place on their sites.
+
+Plausible Analytics script weights less than 1 KB, which is 45 times smaller than the Google Analytics Global Site Tag script. Plausible will reduce your page weight, keep your site loading fast and reduce your electricity usage.
+
+A site with 10,000 monthly visitors can save more than 4.5 kg of CO2 emissions per year by replacing Google Analytics with Plausible Analytics. And with Plausible now installed on more than 10,000 sites and counting hundreds of millions of page views every month, this adds up. You can read more about what makes Plausible a [lightweight web analytics](https://plausible.io/lightweight-web-analytics) solution.
+
+The environmental causes and projects we will donate to remain to be decided. We're thinking of wildlife preservation, clean water and clean air initiatives. We'll take time later this year to pick out projects that make the most sense to us.
+
+### Open source software projects 
+
+On the open source side of things, we want to donate to the projects we stand on the shoulders of. The full list of projects remains to be decided.
+
+Running an open source project, we're very aware of how tough it is to [pay your rent](https://plausible.io/blog/open-source-funding) when the only accepted funding model is donations. Maintaining open source software is a thankless, time-consuming job. We released our code on GitHub and made it easy to self-host on principle, not because it's a profitable business.
+
+We had three $5/month donations for Plausible self-hosted in the first six months of accepting contributions and in the three months or so since that, I've been on a personal quest to try and improve that number.
+
+We're now up on nine $5/month donations and we're grateful to the people who decide to donate. Each donation helps contribute back to the Plausible community and the sustainability of our project.
+
+The fact is that it's easier to find a Google Analytics user, tell them about Plausible, let them have the trial and become a paying subscriber than it is to get a person who's self-hosting Plausible to [donate to the long-term sustainability of the project](https://github.com/sponsors/plausible). 
```

Comment:
I feel like this section contradicts itself. At the same time we say donations are a bad model but also that we will start participating in this model  :)

I think donations are a bad model _for us_ and for other standalone SaaS products. But there are other projects that are developing foundational developer tools like languages, databases, etc. Nobody pays for using a programming language for example, the ecosystem for this just doesn't exist. Developers like @lpil and organizations like [Erlang Ecosystem Foundation](https://erlef.org/) rely on donations to do R&D for new languages. We stand on the shoulders of these giants and for better or worse, these are almost exclusively donation-funded projects.

I would frame it as "we're lucky to have such a straight-forward model to monetize. Many open source projects that we rely on don't have such a simple way to fund their activities."

Reply:
💜